x-pack/plugins/file_data_visualizer/public/application/components/examples_list/examples_list.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+
+import { EuiListGroup, EuiListGroupItem } from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import { ExpandedRowFieldHeader } from '../stats_table/components/expanded_row_field_header';
+interface Props {
+  examples: Array<string | object>;
+}
+
+export const ExamplesList: FC<Props> = ({ examples }) => {
+  if (examples === undefined || examples === null || !Array.isArray(examples)) {
+    return null;
+  }
+  let examplesContent;
+  if (examples.length === 0) {
+    examplesContent = (
+      <FormattedMessage
+        id="xpack.fileDataVisualizer.fieldDataCard.examplesList.noExamplesMessage"
+        defaultMessage="No examples were obtained for this field"
+      />
+    );
+  } else {
+    examplesContent = examples.map((example, i) => {
+      return (
+        <EuiListGroupItem
+          className="mlFieldDataCard__codeContent"
+          size="s"
+          key={`example_${i}`}
+          label={typeof example === 'string' ? example : JSON.stringify(example)}
+        />
+      );
+    });
+  }
+
+  return (
+    <div data-test-subj="mlFieldDataExamplesList">
```

Comment:
How about changing mlFieldDataExamplesList to fieldDataExamplesList?

Reply:
i'm reverting this change as it breaks functional tests. We have common test code for the index and file data visualizers which will have to be changed, but that will be quite a bit of work. 
so for the sake of getting this PR in before feature freeze it think it's ok to have some incorrect `data-test-subj` labels in this PR